Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -105,7 +127,7 @@ InModuleScope MSFT_xIPAddress {
 
             It 'should be $true' {
                 $Splat = @{
-                    IPAddres = '192.168.0.1'
+                    IPAddress = '192.168.0.1'
                     InterfaceAlias = 'Ethernet'
                 }
                 $Result = ValidateProperties @Splat
```

Comment:
Can you point me in the right direction for how to do that? I'm completely new to the pester tests. I just need a good example that I can copy.